app/models/solidus_subscriptions/subscription.rb

```diff
@@ -432,5 +432,14 @@ def emit_events_for_update
         emit_event(type: 'subscription_payment_method_changed')
       end
     end
+
+    def self.ransackable_attributes(_auth_object = nil)
+      %w[actionable_date billing_address_id created_at currency end_date guest_token id id_value interval_length interval_units paused payment_method_id payment_source_id payment_source_type shipping_address_id skip_count state store_id successive_skip_count updated_at user_id]
```

Comment:
I think guest_token should not be searchable. I also see no value in some of the other attributes. Can we just add the attributes that are used by admins in the search field?

Reply:
Thanks. I have checked and added the needed searchable attributes only now.